src/views/admin/UserInfo.tsx

```diff
@@ -96,8 +109,12 @@ const UserInfo = () => {
           </div>
 
           <div>
-            <CustomButton variant="contained" className=" mt-2" type="submit">
-              Submit
+            <CustomButton
+              variant="contained"
+              className=" min-w-full mt-2"
+              type="submit"
+            >
+              {isLoading ? <Loader /> : "Submit"}
```

Comment:
Just call the CircularProgress component here instead of the loader

Reply:
Fixed, raising new PR